etc/spack/defaults/concretizer.yaml

```diff
@@ -89,3 +89,7 @@ concretizer:
   # Static analysis may reduce the concretization time by generating smaller ASP problems, in
   # cases where there are requirements that prevent part of the search space to be explored.
   static_analysis: false
+
+  concretization_cache:
+    enable: true
+    url: $misc_cache/concretization
```

Comment:
misc_cache is not a thing, the cache is getting dumped to `etc/spack/defaults/\$misc_cache/concretization/...`

Reply:
Fixed